### ISSUE TITLE
Incompatible with SonarQube 8.5 and above

### ISSUE THREAD

**Issue by michaelsauter** [opendevstack/ods-core#960]
**Describe the bug**
When using SonarQube 8.5 or above, the server stops with the following error:
```
Web server startup failed: The following plugin is no longer compatible with this version of SonarQube: 'scmgit'
```

Apparently this plugin functionality is now embedded: https://github.com/SonarSource/sonar-scm-git#enbedded-since-sonarqube-85

**Affected version:**
 - OpenDevStack: master, 3.x

**Additional context**
The `ods-core.env.sample` file only states compatibility with up to 8.2, so this sort of thing is to be expected. However, we should be compatible with newer versions going forward.

**Comment by michaelsauter** [comment]
Actually, https://github.com/cnescatlab/sonar-cnes-report/issues/172 prevents us from moving to 8.5 as well. We should note in the config that 8.4 is the latest supported version. FYI @segator @borja44 @felipecruz91

**Comment by michaelsauter** [comment]
I just checked and cues-report 4.0.0 was released bringing compatibility with SQ 8.9 LTS: https://github.com/cnescatlab/sonar-cnes-report/releases/tag/4.0.0

**Comment by gerardcl** [comment]
will be fixed by https://github.com/opendevstack/ods-core/pull/1096 - that will come in ODS 4.2.0 release
